src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -82,6 +86,44 @@ private static OngoingTaskReplication GetReplicationTaskInfo(ClusterTopology clu
                 DestinationURL = replicationNode.Url,
             };
         }
+
+        [RavenAction("/admin/update-watcher", "POST", "/admin/update-watcher?name={databaseName:string}")]
+        public async Task UpdateWatcher()
+        {
+            var name = GetQueryStringValueAndAssertIfSingleAndNotEmpty("name");
+
+            if (ResourceNameValidator.IsValidResourceName(name, ServerStore.Configuration.Core.DataDirectory.FullPath, out string errorMessage) == false)
+                throw new BadRequestException(errorMessage);
+
+            ServerStore.EnsureNotPassive();
```

Comment:
Don't think this is needed, this require a db to already exists, so we can't still be passive here.